Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/QuaternionTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class QuaternionTests : TypeTester<Quaternion>
+    {
+        public static (Quaternion deserialized, string serialized)[] representations = new[] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4930242)